src/main/java/racingcar/Names.java

```diff
@@ -0,0 +1,36 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class Names {
+
+  private final ArrayList<String> names;
+
+  public Names(String unsplitted) {
```

Comment:
```suggestion
  public Names(String[] splitted) {
```

`","`까지 알게 하는게 좋을까요? 아니면 그냥 분리된 배열을 주는게 좋을까요?
정답은 없습니다. 저는 고민이 되네요 😄